include/rmm/mr/device/cuda_async_memory_resource.hpp

```diff
@@ -47,8 +47,9 @@ class cuda_async_memory_resource final : public device_memory_resource {
     RMM_CUDA_TRY(cudaGetDevice(&device));
     int v{0};
     auto e = cudaDeviceGetAttribute(&v, cudaDevAttrMemoryPoolsSupported, device);
-    RMM_EXPECTS(e == cudaSuccess && v == 1,
-                "cudaMallocAsync not supported with this CUDA driver/runtime version");
+    RMM_EXPECTS(
+      e == cudaSuccess && v == 1,
+      "cudaMallocAsync not supported by the version of the CUDA Toolkit used for this build");
```

Comment:
This is not the error you want to change!  if `RMM_CUDA_MALLOC_ASYNC_SUPPORT` is true, then `cudaMallocAsync` *is* supported by the CUDA toolkit used for this error. But this is a runtime error which means that the driver/runtime being run on doesn't support it. You want to change the error message in the `#else` branch.

Reply:
Fixed (accepted your suggestions!) -  thanks

Reply:
I think this error message is wrong now, and it should be reverted?

Reply:
+1

Reply:
Fixed now